mc2.model.csv

```diff
@@ -47,6 +47,136 @@ Resource Access Hazard,"Sensory hazards (Flashing, Motion, etc.) applicable to t
+Filename,Name of a file,,,TRUE,,,,HTAN,
+File Format,"Format of a file (e.g. txt, csv, fastq, bam, etc.)",,,TRUE,,,,HTAN,
+Parent Biospecimen ID,Biospecimen Identifier indicating the biospecimen(s) from which these files were derived; multiple parent biospecimen should be comma-separated,,,TRUE,,,,HTAN,
+Synapse ID of Data File,Synapse ID of file,,,TRUE,,,,HTAN,
+Synapse ID of GeoMx Lab Worksheet File,Synapse ID(s) of Lab Worksheet Files output from the GeoMx DSP workflow. Multiple files are listed as comma separated values.,,,FALSE,,,,HTAN,
+Synapse ID of GeoMx DSP PKC File,The Synapse ID(s) associated with the PKC mapping file for the assay. Multiple files are listed as comma separated values.,,,FALSE,,,,HTAN,
+Synapse ID of GeoMx DSP ROI Segment Annotation File,Synapse ID(s) for ROI/AOI/Segmentation annotations in the GeoMx DSP experiment.,,,FALSE,,,,HTAN,
+GeoMx DSP Assay Type,The assay type which was used for the GeoMx DSP pipeline.,,,FALSE,,,,HTAN,
+Target Depth,The targeted read depth prior to sequencing. Integer,,,TRUE,,,,HTAN,
+Read Length,"The length of the sequencing reads. Can be integer, null",,,TRUE,,,,HTAN,
+ROI Name,Name of Region of Interest corresponding to file. Multiple names should be comma-separated,,,TRUE,,,,HTAN,
+AOI Name,Name of Area of Illumination corresponding to file. Multiple names should be comma-separated,,,TRUE,,,,HTAN,
+Software and Version,Name of software used to generate expression values. String,,,FALSE,,,,HTAN,
+GeoMx DSP NGS Sequencing Platform,A platform is an object aggregate that is the set of instruments and software needed to perform a process [OBI_0000050]. Specific model of the sequencing instrument.,,,FALSE,,,,HTAN,
+GeoMx DSP NGS Library Selection Method,How RNA molecules are isolated.,,,FALSE,,,,HTAN,
+GeoMx DSP NGS Library Preparation Kit Name,Name of Library Preparation Kit. String,,,FALSE,,,,HTAN,
+GeoMx DSP Library Preparation Kit Vendor,Vendor of Library Preparation Kit. String,,,FALSE,,,,HTAN,
+GeoMx DSP Library Preparation Kit Version,Version of Library Preparation Kit. String,,,FALSE,,,,HTAN,
+NanoString GeoMx DSP Spatial Transcriptomics Level 2,Raw counts (DCC/RCC) files from GeoMX experiments,,"Component, Filename, File Format, Parent Biospecimen ID, Synapse ID of Data File, Synapse ID of Parent FASTQ File, Synapse ID of GeoMx Lab Worksheet File, Synapse ID of GeoMx DSP PKC File, Synapse ID of GeoMx DSP ROI Segment Annotation File, GeoMx DSP Assay Type, ROI Name, AOI Name, Software and Version, GeoMx DSP NGS Sequencing Platform, GeoMx DSP NGS Library Selection Method, GeoMx DSP NGS Library Preparation Kit Name, GeoMx DSP Library Preparation Kit Vendor, GeoMx DSP Library Preparation Kit Version",FALSE,,,,HTAN,
```

Comment:
> A "summary.txt" file is expected which aggregates all DCC files.

This seems reasonable to collect, I agree. I think it will be good to have a manifest for individual DCC/RCC file metadata and a manifest for the "summary.txt" available, to accommodate either approach.

>Similarly, a config.ini file should be expected to be uploaded.

This makes sense for the fastq.gz/DCC route, but I don't think this applies to RCC files, so we will need to account for this in the model.